doc/release/RELEASE-NOTES.md

```diff
@@ -19,6 +23,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### What's new
 
 ### Release instructions:
+* Recreate index with new schema:
+    1. Before deploying to the env, especially for Production and Staging, you must first delete the opensearch production location index through EС2 bastion.
+    2. Delete the EFS storage from the ran file or the file through the EС2 bastion instance.
```

Comment:
What is "from the ran file"? What does it mean?

Reply:
I meant the running Logstash app. Will rename it.

Reply:
Then:
Delete the appropriate EFS storage or the file saved on the EFS storage through the EC2 bastion instance.